pom.xml

```
@@ -1540,7 +1540,7 @@
 
     <!-- NOTE: When updating GeoTools, check which version
     of Postgres is used and update pg.version if needed. -->
-    <geotools.version>27.1</geotools.version>
+    <geotools.version>28-RC</geotools.version>
```

This needs to be changed to 28.0 when it's released.

---

@davidblasby, version 28 it's released, please if you can apply this change:

```suggestion
    <geotools.version>28.0</geotools.version>
```